NETWORK/NetworkCreateSynchronisedScene.md

```diff
@@ -7,20 +7,39 @@ ns: NETWORK
 // 0x7CD6BC4C2BBDD526 0xB06FE3FE
 int NETWORK_CREATE_SYNCHRONISED_SCENE(float x, float y, float z, float xRot, float yRot, float zRot, int rotationOrder, BOOL holdLastFrame, BOOL looped, float p9, float animTime, float animSpeed);
 ```
-
 ## Parameters
-* **x**: 
-* **y**: 
-* **z**: 
-* **xRot**: 
-* **yRot**: 
-* **zRot**: 
-* **rotationOrder**: 
-* **holdLastFrame**: 
-* **looped**: 
-* **p9**: 
-* **animTime**: 
-* **animSpeed**: 
+* **x**: The x value of the entity you want to do the animation on.
+* **y**: The y value of the entity you want to do the animation on.
+* **z**: The z value of the entity you want to do the animation on.
+* **xRot**: The xRotation value of the entity you want to do the animation on.
+* **yRot**: The yRotation value of the entity you want to do the animation on.
+* **zRot**: The zRotation value of the entity you want to do the animation on.
+* **rotationOrder**: Rotation order.
+* **holdLastFrame**: Whether the animation ends (smthn like ClearPedTask) if true, or doesn't end till you make it. 
+* **looped**: Whether the animation is looping. 
+* **p9**: a big number like 100000000000 (doesn't seem to affect anything).
+* **animTime**: I think 0 for default animation time, or some number in msec.
+* **animSpeed**: A big number 1000000000 (also doesn't seem to affect anything).
+
+### Rotation Orders
+* **0**: ZYX - Rotate around the z-axis, then the y-axis and finally the x-axis.
+* **1**: YZX - Rotate around the y-axis, then the z-axis and finally the x-axis.
+* **2**: ZXY - Rotate around the z-axis, then the x-axis and finally the y-axis.
+* **3**: XZY - Rotate around the x-axis, then the z-axis and finally the y-axis.
+* **4**: YXZ - Rotate around the y-axis, then the x-axis and finally the z-axis.
+* **5**: XYZ - Rotate around the x-axis, then the y-axis and finally the z-axis.
+
+## Used With
+* **NetworkAddPedToSynchronisedScene**: https://docs.fivem.net/natives/?_0x742A637471BCECD9.
+* **NetworkAddEntityToSynchronisedScene**: https://docs.fivem.net/natives/?_0xF2404D68CBC855FA.
+* **NetworkStartSynchronisedScene**: https://docs.fivem.net/natives/?_0x9A1B3FCDB36C8697.
```

Comment:
The rotation orders section should probably make reference to another native (e.g., from the parameter description or something). Unless you want to be consistent and have 35 other natives copy and paste the same (and still incorrect) list.

Reply:
> The rotation orders section should probably make reference to another native (e.g., from the parameter description or something). Unless you want to be consistent and have 34 other natives copy and paste the same (and still incorrect) list.

Yep, to avoid redundancy. I did this for another [native](https://github.com/citizenfx/natives/pull/866/files) that had the same example copy-pasted over and over. Better to centralize and just reference 👍

Reply:
> This should all be above the `## Parameters` section. Make sure to use triple hashes `###` for headers like you did with Rotation Orders but for 'Used With', double hashes are not allowed for custom headers 😊

By the way, I initially overlooked this detail, but rotation orders are located [here](https://github.com/citizenfx/natives/blob/462ec333001ce00a9c9d420b58b9cf48dda19426/ENTITY/GetEntityRotation.md?plain=1#L21), just a reference as in:

```
You can find the rotation orders in the following native named: [`GET_ENTITY_ROTATION`](#_0xAFBD61CC738D9EB9).
```

It's better for everyone to avoid redundancy, that way if information is wrong/needs to be updated, it only needs to be done at a specific location.

Thanks!